src/main/java/de/tum/in/www1/artemis/service/linkpreview/LinkPreviewService.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.in.www1.artemis.service.linkpreview;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.service.linkpreview.ogparser.Content;
+import de.tum.in.www1.artemis.service.linkpreview.ogparser.OgParser;
+import de.tum.in.www1.artemis.service.linkpreview.ogparser.OpenGraph;
+import de.tum.in.www1.artemis.service.linkpreview.ogparser.htmlparser.OgMetaElementHtmlParser;
+import de.tum.in.www1.artemis.web.rest.dto.LinkPreviewDTO;
+
+/**
+ * Service for retrieving meta information from a given url.
+ */
+@Service
+public class LinkPreviewService {
```

Comment:
Do we need a service here? Looks like the method could just be `static` and we can make this a util instead of a service

Reply:
I thought that in the future this can be extended to support oEmbed server and twitter card meta tags and the service here makes sense but i did not think of utils. What do you suggest ?